pytorch_blade/src/BUILD

```diff
@@ -64,9 +76,9 @@ pybind_library(
         "//visibility:public",
     ],
     deps = [
-        ":torch_blade",
+        ":torch_blade_shared",
```

Comment:
Just curious if this `pybind_library` could depend on target `libtorch_blade.so` directly?